tools/export/cces/__init__.py

```diff
@@ -108,14 +108,14 @@ def format_src_path(path):
         return CCES.format_path(path, "PARENT-1-PROJECT_LOC/")
 
     @staticmethod
-    def clean_flags(container, flags):
+    def clean_flags(container, flags_to_remove):
         """
         Some flags are handled by CCES already, so there's no need
         to include them twice.
         """
-        for flag in container:
-            if flag in flags:
-                flags.remove(flag)
+        for flag in flags_to_remove:
+            if flag in container:
+                container.remove(flag)
```

Comment:
Could you split this rename into another commit?

Reply:
For my own education, could you clarify why it should be a separate commit? Thanks!

Reply:
This change (as far as I can tell) is unrelated. I would like to keep the changes related to the fix separate from format/naming changes.

Reply:
The rename was added because the bug was caused by a confusion over what the two arguments referred to. By renaming the arguments it should avoid such a confusion in the future. You'll notice that there was more change here than just the rename; the container flags are now removed instead of the flags_to_remove flags

Reply:
:+1: Thanks for the explanation!